bentoml/_internal/cli/yatai.py

```diff
@@ -41,12 +42,15 @@ def login(endpoint: str, api_token: str) -> None:
 
         if user is None:
             raise CLIException("current user is not found")
+        ctx.email = user.email if user.email is not None else None
```

Comment:
Does the user attrs class already set missing email files to None? If so we don’t need this check

Reply:
oh we already set the email to None

Reply:
then we can remove this line right?

Reply:
we just need to set ctx.emaill to user.email

Reply:
Going to revert back to original implementation